draft-ietf-httpbis-priority.md

```diff
@@ -666,11 +666,15 @@ about the requests it initiates.
 
 # Server Scheduling
 
-Priority signals are input to a prioritization process. They do not guarantee
-any particular processing or transmission order for one response relative to any
-other response. An endpoint cannot force a peer to process concurrent request in
-a particular order using priority. Expressing priority is therefore only a
-suggestion.
+Server scheduling is a prioritization process, based on many inputs and factors,
+that determine how connection resources such as bandwidth are used. This section
+describes considerations that servers can make when scheduling.
+
+Priority signals are only one form of input to a prioritization process. They do
```

Comment:
How about something like:
```suggestion
It is generally beneficial for an HTTP server to send all responses as early as
possible. However, when serving multiple requests on a single connection, there
could be competition between the requests for resources such as connection
bandwidth. This section describes considerations regarding how servers can
schedule the order in which the competing responses will be sent, when such
competition exists.

Server scheduling is a prioritization process based on many inputs and factors,
with priority signals being only one form of input. Priority signals do
```

Regarding #1794, I think there are two paths forward. One is to explain how responses should be served in general, and the other is to limit the scope of this section to how to prioritize one above another when there is competition.

The change proposed in this comment takes the latter approach, while pointing _out as a fact_ that generally speaking serving every response ASAP is beneficial (hence subsuming the new paragraph being proposed below).

Reply:
I like this!